### PR TITLE
hwdef for Kakute F4 Mini

### PR DESCRIPTION
**Pull request by andyp1per**

### CODE REVIEW

**Comment by peterbarker** [comment]
Tested on HW?

**Comment by andyp1per** [comment]
Well it boots, runs motors and there are no obvious errors. I am seeing if someone on the forums wants to try it as I don't have it in a quad and don't really have time to solder everything up to try.

**Comment by andyp1per** [comment]
I have verified ESC telemetry and BDShot. I will try and verify RCIN before the call.

**Comment by andyp1per** [comment]
RCIN is good

File: Tools/AP_Bootloader/board_types.txt [hunk 1]
```diff
@@ -79,6 +79,7 @@ AP_HW_MRO_CONTROL_ZERO_OEM            142
 AP_HW_MATEKF765_WING                  143
 AP_HW_JDMINIF405                      144
 AP_HW_KAKUTEF7_MINI                   145
+AP_HW_KAKUTEF4_MINI                   146
```
Comment (tridge) [inline Tools/AP_Bootloader/board_types.txt:82]: why this ID? why not one in the ardupilot range?

**Review by tridge — CHANGES_REQUESTED** [review]

File: Tools/AP_Bootloader/board_types.txt (hunk 1 above)
Comment (andyp1per) [reply Tools/AP_Bootloader/board_types.txt]: Sorry did not know that range was reserved, fixed now

**Review by tridge — APPROVED** [review]

**Pull request merged**